src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,72 @@
+package duke;
+
+import commands.*;
+import exceptions.DukeException;
+import tasks.*;
+
+public class Parser {
+
+    public static CommandType getCommandType(String input) {
+        for (CommandType c : CommandType.values()) {
+            if (c.equals(input)) {
+                return c;
+            }
+        }
+        return CommandType.UNKNOWN;
+    }
+
+    public static boolean isDescriptionEmpty(String[] arr) {
+        return arr.length == 1 || arr[1].trim().isEmpty();
+    }
+
+    public static Command parse(String input) throws DukeException {
+        String[] inputArr = input.split(" ", 2);
```

Comment:
Perhaps we can better understand this String array variable by naming it `inputs` or `inputTokens`?